### DISCUSSION TITLE
On CSV Import, receive error "Title required by GitHub, but not found in CSV."

### DISCUSSION THREAD

**Discussion by jrsteensen** [gavinr/github-csv-tools#93] [Help] [answered]
When I try to import the attached CSV, I receive an error "Title required by GitHub, but not found in CSV."

Is this CSV malformed somehow?

![image](https://user-images.githubusercontent.com/2449496/189702437-84a4184c-39cd-4ab3-8f45-56a5577069f8.png)
![image](https://user-images.githubusercontent.com/2449496/189702639-a914cd3d-b76f-4609-be5f-0c0a3c345e15.png)

[OH_drawing_issues.csv](https://github.com/gavinr/github-csv-tools/files/9549809/OH_drawing_issues.csv)

**Comment by jrsteensen** [comment]
I tried it with the test csvs as well, and got the same thing. Not sure how to get a debug output on it, since it seems like its just not building the index on title, even though title is there.

**Comment by slam-campup** [comment]
The first row of my csv file looks like this:
`title,body,labels,milestone`

Is yours the same?

**Comment by fabianchua6** [comment, marked as answer]
I tried using the `1.csv` test file, it works when I used `title, body, labels`  All the body has to be filled though.
